### ISSUE TITLE
examples not include default units for lat, lon, etc

### ISSUE THREAD

**Issue by crotwell** [FDSN/StationXML#48]
in this [example](https://github.com/FDSN/StationXML/blob/draft/docs/examples/kinemetrics_etna_fba-3.xml), the units are given even though there is a default in the schema. It is overly verbose and I don't think this is a best practice.

It also uses non-SI unit names, which I think should also not be done in the examples.

```
      <Latitude unit="DEGREES">34.945911</Latitude>
      <Longitude unit="DEGREES">-106.457199</Longitude>
      <Elevation unit="METERS">1820.0</Elevation>
```

**Comment by crotwell** [comment]
Note also that for attributes that are `fixed`, the value cannot be changed, so specifying it is either redundant or wrong.

From W3C xschema spec:
> default specifies that the attribute is to appear unconditionally in the ·post-schema-validation infoset·, with the supplied value used whenever the attribute is not actually present; 
> fixed indicates that the attribute value if present must equal the supplied constraint value, and if absent receives the supplied value as for default.

**Comment by crotwell** [comment]
fixed in PR #83
